core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/drop.kt

```diff
@@ -0,0 +1,47 @@
+package org.jetbrains.kotlinx.dataframe.api
+
+import org.jetbrains.kotlinx.dataframe.samples.api.firstName
+import org.jetbrains.kotlinx.dataframe.samples.api.lastName
+import org.jetbrains.kotlinx.dataframe.samples.api.name
+import org.junit.Test
+
+class DropTests : ColumnsSelectionDslTests() {
+
+    @Test
+    fun `drop and dropLast`() {
+        listOf(
+            df.select { name.lastName },
+            df.select { name.dropCols(1) },
+            df.select { "name".dropCols(1) },
+            df.select { Person::name.dropCols(1) },
+            df.select { pathOf("name").dropCols(1) },
+        ).shouldAllBeEqual()
+
+        listOf(
+            df.select { name.firstName },
+            df.select { name.dropLastCols(1) },
+            df.select { "name".dropLastCols(1) },
+            df.select { Person::name.dropLastCols(1) },
+            df.select { pathOf("name").dropLastCols(1) },
+        ).shouldAllBeEqual()
+    }
+
+    @Test
+    fun `dropWhile and dropLastWhile`() {
+        listOf(
+            df.select { name.lastName },
+            df.select { name.dropColsWhile { it.name == "firstName" } },
+            df.select { "name".dropColsWhile { it.name == "firstName" } },
```

Comment:
I'm starting to think that "While" postfix for working with Predicates is not so good here and in another functions.
What do you think? Is dropLastColsFilter or dropLastColsIf would be better? Looking at `while` I'm starting to think about cycles (it has predicates too, I agree).

Reply:
just following standard names: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/drop-while.html